apps/web/src/components/Calendar/calendar.module.scss

```diff
@@ -0,0 +1,34 @@
+.discussion {
```

Comment:
Have we coordinated about having these colors integrated with the themes? @brodeur did these get added to the themes? Do they shift based on the theme mode?